src/Common/src/Interop/Windows/mincore/Interop.Manual.cs

```diff
@@ -0,0 +1,266 @@
+        internal const int ERROR_SHARING_VIOLATION = 0x20;
+        internal const int ERROR_NOT_SUPPORTED = 0x32;
+        internal const int ERROR_FILE_EXISTS = 0x50;
+        internal const int ERROR_INVALID_PARAMETER = 0x57;
+        internal const int ERROR_BROKEN_PIPE = 0x6D;
+        internal const int ERROR_CALL_NOT_IMPLEMENTED = 0x78;
+        internal const int ERROR_INSUFFICIENT_BUFFER = 0x7A;
+        internal const int ERROR_INVALID_NAME = 0x7B;
+        internal const int ERROR_BAD_PATHNAME = 0xA1;
+        internal const int ERROR_ALREADY_EXISTS = 0xB7;
+        internal const int ERROR_ENVVAR_NOT_FOUND = 0xCB;
+        internal const int ERROR_FILENAME_EXCED_RANGE = 0xCE;  // filename too long.
+        internal const int ERROR_NO_DATA = 0xE8;
+        internal const int ERROR_PIPE_NOT_CONNECTED = 0xE9;
+        internal const int ERROR_MORE_DATA = 0xEA;
+        internal const int ERROR_DIRECTORY = 0x10B;
+        internal const int ERROR_OPERATION_ABORTED = 0x3E3;  // 995; For IO Cancellation
+        internal const int ERROR_NOT_FOUND = 0x490;          // 1168; For IO Cancellation
+        internal const int ERROR_NO_TOKEN = 0x3f0;
+        internal const int ERROR_DLL_INIT_FAILED = 0x45A;
+        internal const int ERROR_NON_ACCOUNT_SID = 0x4E9;
+        internal const int ERROR_NOT_ALL_ASSIGNED = 0x514;
+        internal const int ERROR_UNKNOWN_REVISION = 0x519;
+        internal const int ERROR_INVALID_OWNER = 0x51B;
+        internal const int ERROR_INVALID_PRIMARY_GROUP = 0x51C;
+        internal const int ERROR_NO_SUCH_PRIVILEGE = 0x521;
+        internal const int ERROR_PRIVILEGE_NOT_HELD = 0x522;
+        internal const int ERROR_NONE_MAPPED = 0x534;
+        internal const int ERROR_INVALID_ACL = 0x538;
+        internal const int ERROR_INVALID_SID = 0x539;
+        internal const int ERROR_INVALID_SECURITY_DESCR = 0x53A;
+        internal const int ERROR_BAD_IMPERSONATION_LEVEL = 0x542;
+        internal const int ERROR_CANT_OPEN_ANONYMOUS = 0x543;
+        internal const int ERROR_NO_SECURITY_ON_OBJECT = 0x546;
+        internal const int ERROR_TRUSTED_RELATIONSHIP_FAILURE = 0x6FD;
+
+        // Error codes from ntstatus.h
+        internal const uint STATUS_SUCCESS = 0x00000000;
+        internal const uint STATUS_SOME_NOT_MAPPED = 0x00000107;
+        internal const uint STATUS_NO_MEMORY = 0xC0000017;
+        internal const uint STATUS_OBJECT_NAME_NOT_FOUND = 0xC0000034;
+        internal const uint STATUS_NONE_MAPPED = 0xC0000073;
+        internal const uint STATUS_INSUFFICIENT_RESOURCES = 0xC000009A;
+        internal const uint STATUS_ACCESS_DENIED = 0xC0000022;
+
+        // PInvokes
+        [DllImport("api-ms-win-core-handle-l1-1-0", SetLastError = true)]
+        internal static extern bool CloseHandle(IntPtr handle);
+
+        [DllImport("api-ms-win-core-heap-obsolete-l1-1-0", SetLastError = true)]
+        internal static extern IntPtr LocalFree(IntPtr handle);
```

Comment:
We already have CloseHandle and LocalFree defined in Interop.  Please reuse all of the existing DllImport definitions where possible rather than adding new ones.